cmd/ocm/gcp/scripting.go

```diff
@@ -263,8 +264,19 @@ func addRoleBindingsScript(wifConfig *cmv1.WifConfig) string {
 			} else {
 				roleResource = fmt.Sprintf("projects/%s/roles/%s", project, role.RoleId())
 			}
-			sb.WriteString(fmt.Sprintf("gcloud projects add-iam-policy-binding %s --member=%s --role=%s\n",
-				project, member, roleResource))
+			if role.ResourceBindings() != nil {
+				for _, resourceBinding := range role.ResourceBindings() {
+					if resourceBinding.Type() != serviceAccountType {
```

Comment:
we should consider atleast printing out an error if its an unexpected/unsupported binding type 
          switch resourceBinding.Type() {
          case "iam.serviceAccounts":
					saResourceId := gcp.FmtSaResourceId(resourceBinding.Name(), project)
					sb.WriteString(fmt.Sprintf("gcloud iam service-accounts add-iam-policy-binding %s --member=%s --role=%s\n",
						saResourceId, member, roleResource))
          default:
              log.Printf("Warning: unsupported resource type '%s' for resource '%s'",
                  resourceBinding.Type(), resourceBinding.Name())
              continue
          }